e2e/run

```diff
@@ -156,13 +163,17 @@ customize_cluster_yaml() {
 worker:
   nodePools:
   - name: asg1
+    clusterAutoscalerSupport:
+      enabled: true
```

Comment:
`clusterAutoscalerSupport` is meant to provide enough permissions to call AWS APIs required to host cluster-autoscaler. Then, probably we should provide a nodeSelector for cluster-autoscaler so that we can ensure CA to be scheduled to nodes with the enough permissions?